src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -149,7 +149,59 @@ private SyntaxTokenList GetParameterModifiers(RefKind refKind)
+        {
+            return CreateConversionDeclaration(parameters, returnType, accessibility, modifiers, statements, isImplicit: true);
+        }
+
+        public override SyntaxNode ExplicitConversionDeclaration(
+            IEnumerable<SyntaxNode> parameters = null, SyntaxNode returnType = null, 
+            Accessibility accessibility = Accessibility.NotApplicable,
+            DeclarationModifiers modifiers = default(DeclarationModifiers), 
+            IEnumerable<SyntaxNode> statements = null)
+        {
+            return CreateConversionDeclaration(parameters, returnType, accessibility, modifiers, statements, isImplicit: false);
+        }
+
+        private SyntaxNode CreateConversionDeclaration(
+            IEnumerable<SyntaxNode> parameters, SyntaxNode returnType, 
+            Accessibility accessibility, DeclarationModifiers modifiers, 
+            IEnumerable<SyntaxNode> statements, bool isImplicit)
+        {
+            return CreateOperator(parameters, returnType, accessibility, modifiers, statements, 
+                (attributes, modifierList, returnTypeNode, parameterList, body, semicolon) =>
+                {
+                    var token = SyntaxFactory.Token(isImplicit ? SyntaxKind.ImplicitKeyword : SyntaxKind.ExplicitKeyword);
+                    return SyntaxFactory.ConversionOperatorDeclaration(
+                       attributes, modifierList, token,
+                       SyntaxFactory.Token(SyntaxKind.OperatorKeyword),
+                       returnTypeNode, parameterList, body, semicolon);
+                });
+        }
+
+        private SyntaxNode CreateOperator(
+            SyntaxKind operatorTokenKind, IEnumerable<SyntaxNode> parameters, SyntaxNode returnType,
+            Accessibility accessibility, DeclarationModifiers modifiers, IEnumerable<SyntaxNode> statements)
+        {
+            return CreateOperator(parameters, returnType, accessibility, modifiers, statements,
+                (attributes, modifierList, returnTypeNode, parameterList, body, semicolon) =>
+                {
+                    return SyntaxFactory.OperatorDeclaration(
+                        attributes, modifierList, returnTypeNode,
+                        SyntaxFactory.Token(SyntaxKind.OperatorKeyword),
+                        SyntaxFactory.Token(operatorTokenKind),
+                        parameterList, body, semicolon);
+                });
+        }
+
+        private delegate SyntaxNode CreateOperatorCallback(
+            SyntaxList<AttributeListSyntax> attributes, SyntaxTokenList modifierList, TypeSyntax returnTypeNode,
+            ParameterListSyntax parameterList, BlockSyntax body, SyntaxToken semicolon);
+
+        private SyntaxNode CreateOperator(
+            IEnumerable<SyntaxNode> parameters, SyntaxNode returnType, Accessibility accessibility,
+            DeclarationModifiers modifiers, IEnumerable<SyntaxNode> statements,
+            CreateOperatorCallback createOperator)
```

Comment:
I don't understand the purpose of this method.  AFAICT, there is only thing ever passed as a `CreateOperatorCallback` - why not just inline that code where it's needed?